lib/repl.js

```diff
@@ -891,6 +892,7 @@ REPLServer.prototype.displayPrompt = function(preserveCursor) {
     const len = this.lines.level.length ? this.lines.level.length - 1 : 0;
     const levelInd = '..'.repeat(len);
     prompt += levelInd + ' ';
+    REPLServer.super_.prototype.undohistory.call(this);
```

Comment:
Minor naming nit: `undohistory` is all lower but other places have camel case, so this would be `undoHistory`.